### ISSUE TITLE
[MU4 Issue] Crash when trying to enter music into last few measures of SATB score using MuseSounds Choir (Win10)

### ISSUE THREAD

**Issue by HuBandiT** [musescore/MuseScore#14598]
**Describe the bug**
The SATB score I freshly created and entered music into, keeps crashing MuseScore when trying to enter the very last few measures of music in the score (the last 2-3 measures of Bass on an SATB score) – even with "playback during editing" turned off.

Opening the score and clicking on a rest in the measures mentioned above results in a crash. If I click on a note in the same measure, that sometimes also results in a crash. If I then enter note entry mode, sometimes I can enter a few more notes and slurs before a crash occurs. Other times a crash results upon typing the key to enter the first note.

This is a fresh score created in MS4. This is my first fresh score I created and edited in MS4.

**To Reproduce**
1. Please open [attached score](https://github.com/musescore/MuseScore/files/10044324/Welcome.Yule.zip)
3. Go to the last few measures in Bass – I do it by dragging the "screen rectangle" in the Navigator panel to show the bottom system on the last page.
4. Click on a note or rest in any measure in the last system (and if that does not trigger a crash, try to enter some notes).
5. A crash occurs.

**Expected behavior**
A crash should not occur, MuseScore should keep working and allow me to enter the final notes of my music.

**Screenshots**
Please see attached video – the video ends (because|at the moment when) MuseScore crashes and its window ceases to exist.

https://user-images.githubusercontent.com/10346452/202788872-ddbfc4b9-3186-4f49-9e4c-1f5d7af24549.mp4

**Platform information**
OS: Windows 10 Version 2009, Arch.: x86_64, MuseScore version (64-bit): 4.0.0-223072007, revision: github-musescore-musescore-2c34155

also happens with MuseScoreNightly-223220403-master-fd02c8d-x86_64.7z

**Additional context**
Intel HD Graphics 4600 integrated GPU

Maybe related to #14422 -> #14324 or #14606 ?

**Comment by Tantacrul** [comment]
Thanks @HuBandiT: does this also occur when you use MS Basic?

I can't reproduce the issue myself (MacOS)

**Comment by HuBandiT** [comment]
Good hunch: no, the issue does not seem to occur with MS Basic:
- Opening the problematic score, I switched to MS Basic and performed about 5-10 music manipulation actions (deleting notes, adding new notes, etc.) and no crash occurred.
- I then quit and relaunched MS, opened the same score (leaving playback MuseSounds), then attempted the same music manipulation actions, at which point a crash occurred again.
- I then relaunched MS, opened the same score, set playback to MS Basic, saved the score, then started manipulating music – no crash.
- Within the same session (with the score open), I switched back to MuseSounds – crash again after a few manipulation actions.
- I then relaunched MS anew, opened the same score (now saved in MS Basic), then attempted to switch to MuseSounds – a crash happened upon clicking the "Activate this profile" button.
- I then relaunched MS anew, opened the same score anew (still saved in MS Basic), then managed to switch to MuseSounds (no crash this time). I then attempted some music manipulation actions – crash again.

Actual playback does not trigger (or only rarely triggers) a crash.

The behaviour does not seem to be entirely deterministic; that is, MS is not crashing at the exact same point each time: sometimes crashes as the first action, other times a few actions – even successfully saving the score some times – are successful, before a crash eventually arrives.
